SamplePlugin/SamplePlugin.csproj

```diff
@@ -125,6 +123,7 @@
     <PropertyGroup>
       <ErrorText>This project references NuGet package(s) that are missing on this computer. Use NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is {0}.</ErrorText>
     </PropertyGroup>
-    <Error Condition="!Exists('..\packages\PlgxTool.0.1.4\build\PlgxTool.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\PlgxTool.0.1.4\build\PlgxTool.targets'))" />
```

Comment:
OK. The tool probably should be modified to exclude `packages.config` automatically like it does with `KeePass.exe`. But, that is  a minor issue.